docker/auto-setup.sh

```diff
@@ -288,15 +288,12 @@ register_default_namespace() {
     else
         echo "Default namespace ${DEFAULT_NAMESPACE} already registered."
     fi
+
+    echo "Waiting for namespace cache to refresh (sleeping for 10 seconds)..."
+    sleep 10
```

Comment:
@pdoerner This seems related to the work you were doing for sync namespace registration. Any way we can get around using a sleep here?

Reply:
Namespace cache should already support read through if namespace is not found. 
I think the problem is there's no read through if search attribute within the namespace config is not found. Then this sleep should happen after search attributes are added? 

An easy fix would be allow specifying search attributes when registering namespace.